package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "grunt-stylelint",
-  "version": "0.10.0",
+  "version": "0.11.0",
   "description": "A Grunt checker for CSS files using Stylelint",
+  "main": "tasks/grunt-stylelint.js",
+  "files": [
+    "tasks"
+  ],
   "scripts": {
-    "test": "grunt test"
+    "test": "node ./node_modules/grunt/bin/grunt test --force"
```

Comment:
Adding `--force` here causes the tests to continue after after any failure and to ignore the failure.

Reply:
This is quiet norma, otherwise we cannot run the test !

Indeed, we have a grunt tasks called stylelint:simple and stylelint:testOutputFile (to check the behavior of the plugin)

And the first task will produce an error report file which will be used for the tests.

````javascript
grunt.registerTask( 'test', [ 'clean:test', 'eslint', 'stylelint', 'nodeunit' ] );
````

So if we do not use "--force", the tests cannot be run

The best approach should be  to create a dedicated Gruntfile.js into the tests and run it when we run the tests (so, with a before and after, clean the generated reports, etc ..) 

But it was not the goal of this PR

Regards